MediaWikiTelegramBot.js

```diff
@@ -1,20 +1,26 @@
+// Require Telegram API & http request
 var TelegramBot = require( 'node-telegram-bot-api' );
 var request = require( 'request' ).defaults( {
     jar: true
 } );
+
+// Require yaml fs reader
 var yaml = require( 'js-yaml' );
 var fs = require( 'fs' );
 
+// So we can use the other functions
 var mwApi = require('./MediaWikiAPI.js');
 
 var mode = 'fetching';
 var apiUrl = 'https://translatewiki.net/w/api.php';
 
 var debugLevel = 1;
 
+// Create object to contain userID and its information
 var userStatus = {};
 var currentMwMessageIndex = 0;
 
+// Debugging is good
```

Comment:
It's good, but you could explain what does this function do differently from simply sending a message. It sends it only if debugLevel is set to 1. (Also, we should move debugLevel to the config file. Could you do it perhaps?)